src/openrct2/ride/VehicleEntry.h

```diff
@@ -93,7 +93,7 @@ enum : uint32_t
 struct rct_ride_entry_vehicle
 {
     uint16_t rotation_frame_mask;
-    uint8_t num_vertical_frames;   // Appears to be unused, except as a temporary variable in RCT2 (not needed for OpenRCT2)
+    uint8_t NumRotationFrames;
```

Comment:
I don't know enough about how it used to be used but would it make sense to have this again split into vert/horizontal? If not I would remove `num_horizontal_frames`.

Reply:
According to the comment it was used as a swap variable or something. I think now we just use allocate a variable on the stack. It's not even read from the .DAT file.

Reply:
Yeah was just wondering if there was any code you noticed in the refactor that had different number of rotation frames in the horizontal compared to the vertical.

Unrelated to this PR but you can see it become a real value that does something in OpenLoco.

Reply:
According to VS "find all references" the only users were the vehicle entry struct and scripting for num_vertical and num_horizontal